dot/parachain/statement-distribution/statement_distribution.go

```diff
@@ -0,0 +1,62 @@
+package statementdistribution
+
+import (
+	"context"
+
+	"github.com/ChainSafe/gossamer/internal/log"
+
+	statementedistributionmessages "github.com/ChainSafe/gossamer/dot/parachain/statement-distribution/messages"
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+)
+
+var logger = log.NewFromGlobal(log.AddContext("pkg", "statement-distribution"))
+
+type StatementDistribution struct {
+}
+
+func (s StatementDistribution) Run(ctx context.Context, overseerToSubSystem <-chan any) {
+
+	for msg := range overseerToSubSystem {
+		err := s.processMessage(msg)
+		if err != nil {
+			logger.Errorf("processing overseer message: %w", err)
+		}
+	}
+}
```

Comment:
Re-write the method the same as other subsystems.

use `select` to receive values from the `overseerToSubSystem` and `ctx.Done()` channels.